CHANGELOG.md

```diff
@@ -1,174 +1,175 @@
 # Changelog
+
 All notable user-facing changes to webknossos are documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
 For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
-
 ## Unreleased
+
 [Commits](https://github.com/scalableminds/webknossos/compare/18.09.0...HEAD)
```

Comment:
Why does this have file have 243 line changes? :see_no_evil: Did a merge go wrong?